README.md

```diff
@@ -1,6 +1,23 @@
-# web3-ui
-This will be a library made up of UI components specifically crafted for web3 use cases.
+# Welcome to web3-ui (The UI library for Web3)
+
+web3-ui is a library made up of UI components specifically crafted for web3 use cases.
 
 [Read the RFC](https://forum.developerdao.com/t/rfc-web3-ui-a-web3-specific-ui-library)
 
+## Installation
+
+web3-ui is available as an [npm package](https://www.npmjs.com/package/web3-ui)
+
+Install using:
+```npm install web3-ui```
+ or
+```npm i web3-ui```
+or
+```yarn add web3-ui```
```

Comment:
```suggestion
```bash
yarn add web3-ui;
# npm i web3-ui;
```

Reply:
Hi I have updated the README based on this suggestion.